diff --git a/wrappers/python/CMakeLists.txt b/wrappers/python/CMakeLists.txt
index 5e8355cad..d2db42d6a 100644
--- a/wrappers/python/CMakeLists.txt
+++ b/wrappers/python/CMakeLists.txt
@@ -281,6 +281,18 @@ add_custom_target(PythonSdist
     COMMENT "Packaging source distribution package (sdist)..."
 )

+# Wheel
+set(PYTHON_SETUP_COMMAND bdist_wheel)
+configure_file(pysetup.cmake.in
+    "${CMAKE_CURRENT_BINARY_DIR}/pysetupbdist_wheel.cmake" @ONLY)
+add_custom_target(PythonBdistWheel
+    COMMAND ${CMAKE_COMMAND}
+        -P "${CMAKE_CURRENT_BINARY_DIR}/pysetupbdist_wheel.cmake"
+    DEPENDS ${STAGING_OUTPUT_FILES} "${CMAKE_CURRENT_BINARY_DIR}/pysetupbdist_wheel.cmake"
+    WORKING_DIRECTORY ${OPENMM_PYTHON_STAGING_DIR}
+    COMMENT "Packaging wheel (bdist_wheel)..."
+)
+
 set(OPENMM_PYTHON_USER_INSTALL OFF CACHE BOOL
     "Whether to install OpenMM Python binary module into the user site-packages directory")
 mark_as_advanced(OPENMM_PYTHON_USER_INSTALL)
diff --git a/wrappers/python/setup.py b/wrappers/python/setup.py
index d2a147065..c7137ec6a 100644
--- a/wrappers/python/setup.py
+++ b/wrappers/python/setup.py
@@ -164,6 +164,7 @@ def buildKeywordDictionary(major_version_num=MAJOR_VERSION_NUM,
     (through custom forces and integrators), openness, and high performance
     (especially on recent GPUs) that make it truly unique among simulation codes.
     """
+    setupKeywords["install_requires"] = ["numpy < 2"]

     define_macros = [('MAJOR_VERSION', major_version_num),
                      ('MINOR_VERSION', minor_version_num)]
