diff --git a/wrappers/python/setup.py b/wrappers/python/setup.py
index ca26e66a5..88f723962 100644
--- a/wrappers/python/setup.py
+++ b/wrappers/python/setup.py
@@ -156,6 +156,7 @@ def buildKeywordDictionary(major_version_num=MAJOR_VERSION_NUM,
                                           "openmm.app" : ['data/*.xml', 'data/*.pdb', 'data/amber14/*.xml', 'data/amber19/*.xml', 'data/charmm36/*.xml', 'data/charmm36_2024/*.xml', 'data/implicit/*.xml'],
                                           "openmm.app.internal" : []}
     setupKeywords["install_requires"]  = ["numpy"]
+    setupKeywords["extras_require"]    = {"cuda12": [f"OpenMM-CUDA-12=={setupKeywords['version']}"], "cuda13": [f"OpenMM-CUDA-13=={setupKeywords['version']}"], "hip6": [f"OpenMM-HIP-6=={setupKeywords['version']}"], "hip7": [f"OpenMM-HIP-7=={setupKeywords['version']}"]}
     setupKeywords["platforms"]         = ["Linux", "Mac OS X", "Windows"]
     setupKeywords["description"]       = \
     "Python wrapper for OpenMM (a C++ MD package)"
